### PR TITLE
Follow indentation in target POM

### PR DESCRIPTION
**Pull request by ia3andy**

### CODE REVIEW

**Comment by ia3andy** [comment]
Ok Updated

File: src/test/java/io/fabric8/maven/merge/SmartModelMergerTest.java [hunk 1]
```diff
@@ -89,4 +93,28 @@ public void should_merge_profiles() {
         assertThat(target.getProfiles().get(0).getModules()).containsExactly("A", "B", "C", "D", "E", "F");
     }
 
+    @Test
+    void should_keep_target_indent_2() throws URISyntaxException {
+        ModelMerger merger = new SmartModelMerger();
+        final Model source = Maven.readModel(Paths.get(getClass().getResource("indent/source-pom.xml").toURI()));
+        final Path targetFile = Paths.get(getClass().getResource("indent/pom-2.xml").toURI());
```
Comment (gastaldi) [inline src/test/java/io/fabric8/maven/merge/SmartModelMergerTest.java:100]: This file is missing from the test

File: src/test/java/io/fabric8/maven/merge/SmartModelMergerTest.java [hunk 2]
```diff
@@ -89,4 +93,28 @@ public void should_merge_profiles() {
...
+        merger.merge(target, source, false, null);
+        Maven.writeModel(target, targetFile);
+        assertThat(targetFile).hasSameBinaryContentAs(Paths.get(getClass().getResource("indent/result-pom-2.xml").toURI()));
+    }
+
+    @Test
+    void should_keep_target_indent_4() throws URISyntaxException {
+        ModelMerger merger = new SmartModelMerger();
+        final Model source = Maven.readModel(Paths.get(getClass().getResource("indent/source-pom.xml").toURI()));
+        final Path targetFile = Paths.get(getClass().getResource("indent/pom-4.xml").toURI());
```
Comment (gastaldi) [inline src/test/java/io/fabric8/maven/merge/SmartModelMergerTest.java:111]: This file is missing from the test

**Review by gastaldi — CHANGES_REQUESTED** [review]

File: src/test/java/io/fabric8/maven/merge/SmartModelMergerTest.java [hunk 3]
```diff
@@ -89,4 +93,28 @@ public void should_merge_profiles() {
...
 
+    @Test
+    void should_keep_target_indent_2() throws URISyntaxException {
+        ModelMerger merger = new SmartModelMerger();
+        final Model source = Maven.readModel(Paths.get(getClass().getResource("indent/source-pom.xml").toURI()));
+        final Path targetFile = Paths.get(getClass().getResource("indent/pom-2.xml").toURI());
+        final Model target = Maven.readModel(targetFile);
+        merger.merge(target, source, false, null);
+        Maven.writeModel(target, targetFile);
+        assertThat(targetFile).hasSameBinaryContentAs(Paths.get(getClass().getResource("indent/result-pom-2.xml").toURI()));
```
Comment (gastaldi) [inline src/test/java/io/fabric8/maven/merge/SmartModelMergerTest.java:104]:
```suggestion
        assertThat(targetFile).hasSameTextualContentAs(Paths.get(getClass().getResource("indent/result-pom-2.xml").toURI()));
```

**Comment by ia3andy** [comment]
Fixes

**Comment by gastaldi** [comment]
@ia3andy this should fix it

**Comment by ia3andy** [comment]
Thanks Mate!!

**Pull request merged**